integration-tests/integration/coordinator/remove-involved-staff.spec.js

```diff
@@ -108,33 +110,33 @@ context('A use of force coordinator can remove involved staff', () => {
       )
   })
 
```

Comment:
Delete commented test?

Reply:
Re-added that test back in 

An issue with cypress is that you can't just run a single test in isolation so I often end up commenting out others in the same suite whilst developing a new test